src/features/views/day/day-carousel/DayItem.module.scss

```diff
@@ -14,6 +14,8 @@
     }
   }
 
+  // DorHakim
```

Comment:
You don't need to put your name everywhere. Git shows which changes you made.
Please remove all ` // DorHakim` comments.